src/App.js

```diff
@@ -1,6 +1,18 @@
-// 해당 위치에 필요한 컴포넌트를 바탕으로 직접 google 검색 페이지를 개발하시면 됩니다!
+import AlbumBox from "./Component/AlbumBox";
+import Footer from "./Component/Footer";
+import MainBox from "./Component/MainBox";
+import NavigationBar from "./Component/NavigationBar";
```

Comment:
컴포넌트의 이름이 아닌 디렉토리명은 PascalCase 대신 camelCase를 사용하는것이 일반적입니다.

```suggestion
import AlbumBox from "./components/AlbumBox";
import Footer from "./components/Footer";
import MainBox from "./components/MainBox";
import NavigationBar from "./components/NavigationBar";
```

위와 같이 변경해볼 수 있을 것 같아요. 
(그리고 아주아주아주 지극히 취향의 영역이지만 저는 단수형보다는 복수형으로 네이밍 하는 것을 선호합니다. 🤣)